### DISCUSSION TITLE
Issue with Intellij: "NoSuchMethodError: HadoopKerberosName.setRuleMechanism"

### DISCUSSION THREAD

**Discussion by EnverOsmanov** [nightscape/spark-excel#649] [Q&A] [answered]
The question is to developers of spark-excel. This is not a bug of spark-excel.
I have some bug with Intellij it fails to run tests for Datareader V2. But if I run tests (`sbt test`) from terminal - there is no such error.

I wonder if anyone else have issue like that?
I'm going to report it to JetBrains, but wanted to gather a bit info if possible.
My environment: Ubuntu 22.04, Coretto 8 JDK, latest `main` branch of spark-excel.

**Comment by nightscape** [comment]
Hi @EnverOsmanov, I never saw that error.
Looks like a dependency version mismatch.
You could try printing out the classpath in the tests and check if they are the same (including the order) when run from sbt vs IntelliJ.

**Reply by EnverOsmanov** [reply]
Do you use Intellij or some other IDE?

**Reply by nightscape** [reply]
Yes, Intellij mostly.

**Comment by EnverOsmanov** [comment, marked as answer]
It seems Intellij's plugin "Big Data Tools" had some confilict. It is working fine now.
